<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PtcPlayerBase</class>
 <widget class="QMainWindow" name="PtcPlayerBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pointcloud Player</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../lib/wombatui_qt/resources/wombat.qrc">
    <normaloff>:/icons/24/media-playback-start.png</normaloff>:/icons/24/media-playback-start.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="margin">
     <number>3</number>
    </property>
    <item>
     <widget class="wombat::GLRenderWidget" name="mRenderWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <widget class="wombat::QPlaybackControl" name="mPlaybackControl" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>120</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>788</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_open"/>
    <addaction name="separator"/>
    <addaction name="action_file_information"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="action_toggle_fullscreen"/>
    <addaction name="action_toggle_coord"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_save_screenshot"/>
    <addaction name="separator"/>
    <addaction name="action_preferences"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_open"/>
   <addaction name="separator"/>
   <addaction name="action_save_screenshot"/>
   <addaction name="separator"/>
   <addaction name="action_toggle_fullscreen"/>
   <addaction name="action_toggle_coord"/>
  </widget>
  <action name="action_open">
   <property name="icon">
    <iconset resource="../../../lib/wombatui_qt/resources/wombat.qrc">
     <normaloff>:/icons/24/document-open.png</normaloff>:/icons/24/document-open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="icon">
    <iconset resource="../../../lib/wombatui_qt/resources/wombat.qrc">
     <normaloff>:/icons/24/system-shutdown.png</normaloff>:/icons/24/system-shutdown.png</iconset>
   </property>
   <property name="text">
    <string>e&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_save_screenshot">
   <property name="icon">
    <iconset resource="../../../lib/wombatui_qt/resources/wombat.qrc">
     <normaloff>:/icons/24/image.png</normaloff>:/icons/24/image.png</iconset>
   </property>
   <property name="text">
    <string>save screenshot</string>
   </property>
  </action>
  <action name="action_preferences">
   <property name="text">
    <string>properties</string>
   </property>
   <property name="toolTip">
    <string>preferences</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>about PTC-Player</string>
   </property>
  </action>
  <action name="action_file_information">
   <property name="text">
    <string>file information</string>
   </property>
  </action>
  <action name="action_toggle_fullscreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../lib/wombatui_qt/resources/wombat.qrc">
     <normaloff>:/icons/24/view-fullscreen.png</normaloff>:/icons/24/view-fullscreen.png</iconset>
   </property>
   <property name="text">
    <string>toggle fullscreen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="action_toggle_coord">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../lib/wombatui_qt/resources/wombat.qrc">
     <normaloff>:/icons/24/3d.png</normaloff>:/icons/24/3d.png</iconset>
   </property>
   <property name="text">
    <string>toggle coordinate system</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>wombat::GLRenderWidget</class>
   <extends>QWidget</extends>
   <header location="global">wombatui_qt/wombatui_qt.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>wombat::QPlaybackControl</class>
   <extends>QWidget</extends>
   <header location="global">wombatui_qt/wombatui_qt.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../lib/wombatui_qt/resources/wombat.qrc"/>
 </resources>
 <connections/>
</ui>
